libs/damap/src/lib/services/backend.service.ts

```diff
@@ -29,202 +37,220 @@ export class BackendService {
+      .pipe(retry(3), catchError(this.handleError('http.error.plans.update')));
   }
 
   getDmpByIdAndRevision(id: number, revision: number): Observable<Dmp> {
-    return this.http.get<Dmp>(`${this.dmpBackendUrl}/${id}/${revision}`).pipe(
-      retry(3),
-      catchError(this.handleError('http.error.versions.revision'))
-    );
+    return this.http
+      .get<Dmp>(`${this.dmpBackendUrl}/${id}/${revision}`)
+      .pipe(
+        retry(3),
+        catchError(this.handleError('http.error.versions.revision'))
+      );
   }
 
   getDmpVersions(id: number): Observable<Version[]> {
-    return this.http.get<Version[]>(`${this.versionBackendUrl}/list/${id}`).pipe(
-      retry(3),
-      catchError(this.handleError('http.error.versions.load'))
-    );
+    return this.http
+      .get<Version[]>(`${this.versionBackendUrl}/list/${id}`)
+      .pipe(retry(3), catchError(this.handleError('http.error.versions.load')));
   }
 
   saveDmpVersion(version: Version): Observable<Version> {
     const httpOptions = {
       headers: new HttpHeaders({
-        'Content-Type': 'application/json'
-      })
+        'Content-Type': 'application/json',
+      }),
     };
-    return this.http.put<Version>(this.versionBackendUrl, version, httpOptions).pipe(
-      retry(3),
-      catchError(this.handleError('http.error.versions.save'))
-    );
+    return this.http
+      .put<Version>(this.versionBackendUrl, version, httpOptions)
+      .pipe(retry(3), catchError(this.handleError('http.error.versions.save')));
   }
 
   getSuggestedProjects(): Observable<Project[]> {
-    return this.http.get<Project[]>(this.projectBackendUrl).pipe(
-      retry(3),
-      catchError(this.handleError('http.error.projects')),
-      shareReplay(1)
-    );
+    return this.http
+      .get<Project[]>(this.projectBackendUrl)
+      .pipe(
+        retry(3),
+        catchError(this.handleError('http.error.projects')),
+        shareReplay(1)
+      );
   }
 
   getProjectMembers(projectId: number): Observable<Contributor[]> {
-    return this.http.get<Contributor[]>(`${this.projectBackendUrl}/${projectId}/staff`).pipe(
-      retry(3),
-      catchError(this.handleError('http.error.projectmembers'))
-    );
-
-  }
-
-  searchPerson(term: string): Observable<Contributor[]> {
-    return this.http.get<Contributor[]>(`${this.backendUrl}persons/search?q=${term}`).pipe(
-      catchError(this.handleError('http.error.person.search'))
-    )
+    return this.http
+      .get<Contributor[]>(`${this.projectBackendUrl}/${projectId}/staff`)
+      .pipe(
+        retry(3),
+        catchError(this.handleError('http.error.projectmembers'))
+      );
   }
 
+  getPersonSearchResult(searchTerm: string, serviceType: string): Observable<Contributor[]> {
+      return this.http
+        .get<Contributor[]>(
+          `${this.backendUrl}persons/search?q=${searchTerm}&searchService=${serviceType}`
+        )
+        .pipe(catchError(this.handleError('http.error.repositories.one')));
```

Comment:
Error message should be `http.error.person.search` not `http.error.repositories.one`.